README.md

```diff
@@ -186,10 +186,13 @@ Use `vector_ip_ops` for inner product and `vector_cosine_ops` for cosine distanc
 Add a vector column
 
 ```python
+from typing import List, Optional
+
 from pgvector.sqlalchemy import Vector
-from sqlalchemy import Column
+from sqlmodel import Column, Field, Session, SQLModel, create_engine, select
 
 class Item(SQLModel, table=True):
+    id: Optional[int] = Field(default=None, primary_key=True)
```

Comment:
Another approach is     

```
id: int = Field(primary_key=True, sa_column_kwargs={"autoincrement": True})
```

They both seem to have same effect, from what I can tell, so maybe something is autoincrementing behind the scenes.